include/eosio/vm/debug_visitor.hpp

```diff
@@ -5,14 +5,14 @@
 
 #include <iostream>
 
-#define DBG_VISIT(name, code)                                                                                          \
+#define DBG_VISIT(name, code, fmt)                                                                                          \
    void operator()(EOS_VM_OPCODE_T(name)& op) {                                                                        \
-      std::cout << "Found " << #name << " at " << get_context().get_pc() << "\n";                                      \
+      std::cout << "Found { " << op.wast() << "} at " << get_context().get_pc() << "\n";                                      \
       interpret_visitor<ExecutionCTX>::operator()(op);                                                                 \
       get_context().print_stack();                                                                                     \
    }
 
-#define DBG2_VISIT(name, code)                                                                                         \
+#define DBG2_VISIT(name, code, fmt)                                                                                         \
```

Comment:
`debug_visitor2` doesn't seem to be used by anything. 

Also both `DBG_VISIT` and `DBG2_VISIT` macros are redundantly `#undef` twice at the bottom of this file.

Reply:
Will leave alone for this PR.